homeworks/taridaniel/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,13 @@
+city = input("Please enter an American city: ").title()
+price = int(input("Please enter the price (USD): "))
+
+if (city == "New York" or city == "San Francisco") and price < 4000:
```

Comment:
Nem hiba, csak rövidítési lehetőség:

`if city in ["New York", "San Francisco"] and price < 4000:`